docs/workflow.md

```diff
@@ -0,0 +1,389 @@
+    }
+  ],
+  "name": [
+    {
+      "family": "Doe",
+      "given": ["John"]
+    }
+  ],
+  "gender": "male",
+  "birthDate": "2000-01-01"
+}
+```
+
+### `Encounter` Resource
+
+The FHIR Encounter resource is used to represent a clinical interaction between a patient and a healthcare provider. It contains information about the patient's visit, such as the location, reason for the visit, and any relevant procedures or diagnoses.
+
+#### `POST ${OPENHIM_ENDPOINT}/encounter`
+
+The `Encounter` resource is an essential part of the LTFU workflow, which is automatically created by the CHT system after a CHW completes the workflow. It triggers FHIR to send a `Subscription` response to the requesting system when there is a match with the `Encounter` resource. This allows for efficient monitoring and follow-up care of patients in the LTFU workflow.
+
+Request
+
+```http
+POST ${OPENHIM_ENDPOINT}/encounter
+
+{
+    "resourceType": "Encounter",
+    "identifier": [
+        {
+            "system": "cht",
+            "value": "${ENCOUNTER_IDENTIFIER}"
+        }
+    ],
+    "status": "finished",
+    "class": "outpatient",
+    "type": [
+        {
+            "text": "Community health worker visit"
+        }
+    ],
+    "subject": {
+        "reference": "Patient/${PATIENT_IDENTIFIER}"
+    },
+    "participant": [
+        {
+            "type": [
+                {
+                    "text": "Community health worker"
+                }
+            ]
+        }
+    ]
+}
+```
+
+Response
+
+```json
+{
+  "resourceType": "Encounter",
+  "id": "5",
+  "meta": {
+    "versionId": "1",
+    "lastUpdated": "2023-04-19T05:00:18.031+00:00"
+  },
+  "identifier": [
+    {
+      "system": "cht",
+      "value": "003b24b5-2396-4d95-bcbc-5a4c63f43ff0"
+    }
+  ],
+  "status": "finished",
+  "type": [
+    {
+      "text": "Community health worker visit"
+    }
+  ],
+  "subject": {
+    "reference": "Patient/3"
+  },
+  "participant": [
+    {
+      "type": [
+        {
+          "text": "Community health worker"
+        }
+      ]
+    }
+  ]
+}
+```
+
+### `Organisation` Resource
+
+The FHIR `Organization` resource represents a group of people or entities with a common purpose or focus. It contains the organization's name, type, and contact details. This resource is often used in healthcare settings to represent healthcare providers, hospitals, clinics, and other organizations involved in patient care. In the LTFU workflow, it represents the **Requesting System**, and it points to its callback URL.
+
+#### `POST ${OPENHIM_ENDPOINT}/organization`
+
+The `Organization` resource in the LTFU workflow represents the Requesting System. Before creating an `Organization`, an `Endpoint` must be created. The `${ORGANIZATION_IDENTIFIER}` is intended to be randomly assigned by the requesting system. It is important to take note of this identifier, as it will be used in future `ServiceRequest`s to identify the requesting system."
```

Comment:
```suggestion
The `Organization` resource in the LTFU workflow represents the Requesting System. Before creating an `Organization`, an `Endpoint` must be created. The `${ORGANIZATION_IDENTIFIER}` is intended to be randomly assigned by the requesting system. It is important to take note of this identifier, as it will be used in future `ServiceRequest`'s to identify the requesting system.
```